woocommerce/class-sv-wc-helper.php

```diff
@@ -548,6 +548,25 @@ public static function is_order_virtual( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Determines if a shop has any published virtual products.
+	 *
+	 * @since 5.10.0
+	 *
+	 * @return bool
+	 */
+	public static function shop_has_virtual_products() {
+
+		$virtual_products = wc_get_products( [
+			'virtual' => true,
+			'status'  => 'publish',
+			'limit'   => - 1,
```

Comment:
could we limit this to `1` instead of unlimited since even just one would make this true?

Reply:
Good point!

Reply:
Tweaked in https://github.com/skyverge/wc-plugin-framework/pull/512/commits/5dd9654bbf4b1957c1d5a1c6b2e88e0f1f227300.